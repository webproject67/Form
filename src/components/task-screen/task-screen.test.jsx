import React from "react";
import renderer from "react-test-renderer";
import TaskScreen from "./task-screen";

it(`Render TaskScreen`, () => {
  const tree = renderer.create(<TaskScreen
    onButtonClickProfile={() => {}}
    onButtonClickStatistics={() => {}}
    buttonClickMenuHandler={() => {}}
    buttonClickArrowHandler={() => {}}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});