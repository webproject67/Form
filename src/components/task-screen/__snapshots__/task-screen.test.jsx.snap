// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render TaskScreen 1`] = `
<main
  className="main"
>
  <nav
    className="main__nav"
  >
    <button
      aria-label="toggle"
      className="main__btn"
      onClick={[Function]}
      type="button"
    />
    <img
      alt="Logo"
      className="main__img"
      src="img/svg/logo.svg"
    />
    <ul
      className="main__list"
    >
      <li
        className="main__item"
      >
        <img
          alt="Logo"
          className="main__image"
          src="img/svg/profile.svg"
        />
        <a
          className="main__link"
          onClick={[Function]}
        >
          Мой профиль
        </a>
      </li>
      <li
        className="main__item"
      >
        <img
          alt="Logo"
          className="main__image"
          src="img/svg/task.svg"
        />
        <a
          className="main__link main__link--active"
        >
          Список задач
        </a>
      </li>
      <li
        className="main__item"
      >
        <img
          alt="Logo"
          className="main__image"
          src="img/svg/statistics.svg"
        />
        <a
          className="main__link"
          onClick={[Function]}
        >
          Статистика
        </a>
      </li>
    </ul>
  </nav>
  <section
    className="main__profile main__profile--width"
  >
    <button
      aria-label="toggle"
      className="main__arrow"
      onClick={[Function]}
      type="button"
    />
    <h2
      className="main__text"
    >
      Список задач
    </h2>
  </section>
</main>
`;
